### PR TITLE
Correct minor spelling and punctuation errors in description file

### PR DESCRIPTION
**Pull request by amadsalmon**
### First time contributor checklist
- [x] I have read the [README](https://github.com/WhisperSystems/Signal-iOS/blob/master/README.md) and [CONTRIBUTING](https://github.com/WhisperSystems/Signal-iOS/blob/master/CONTRIBUTING.md) documents
- [x] I have signed the [Contributor Licence Agreement](https://whispersystems.org/cla/)

### Contributor checklist
- [x] I'm following the [code, UI and style conventions](https://github.com/WhisperSystems/Signal-iOS/blob/master/CONTRIBUTING.md#code-conventions)
- [x] My commits are rebased on the latest master branch
- [x] My commits are in nice logical chunks
- [x] My contribution is fully baked and is ready to be merged as is
- [x] I have tested my contribution on these devices:
 * not applicable

- - - - - - - - - -

### Description
Some minor spelling and punctuation errors were made in the Signal/translations/TRANSLATIONS.md file. I corrected them.

### CODE REVIEW

File: Signal/translations/TRANSLATIONS.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ the same, but in Finnish the noun/verb are distinct.
     /* Button label to archive the current conversation */
     ARCHIVE_ACTION="Archive"
 
-#### Finish
+#### Finnish
```
Comment (Astrodevil) [inline Signal/translations/TRANSLATIONS.md:40]: I think here is no change in spell of finish.   'Finnish' & 'Finish' have different  meanings.

**Review by Astrodevil — APPROVED** [review]

**Review by Astrodevil — COMMENTED** [review]
All minor corrections done here is great. 👍🏼 
Sometimes peoples make mistakes in punctuation & spelling in the lines of code.

File: Signal/translations/TRANSLATIONS.md (hunk 1 above)
Comment (amadsalmon) [reply Signal/translations/TRANSLATIONS.md:40]: True, I don’t know why I was thinking about the language. Thanks for catching that!

Comment (amadsalmon) [reply Signal/translations/TRANSLATIONS.md:40]: Actually, here the author was giving examples about translating to Finnish the language, thus 'Finnish' with two 'n's. You can see that with the strings contained in that subsection: 'Arkisto' is Finnish for 'Archive'.

**Comment by amadsalmon** [comment]
I haven't touched a single line of code. Why doesn't the 'Build and Test' test pass?

**Comment by Astrodevil** [comment]
> I haven't touched a single line of code. Why doesn't the 'Build and Test' test pass?

I think changes you make leaves spacing or blank line may be inappropriate comma or dashes which cause test fails.
I face this issue sometimes, please review your correction once again.

**Pull request closed without merging**